<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entry</class>
 <widget class="QMainWindow" name="Entry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>552</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>642</width>
    <height>552</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="logo">
        <property name="minimumSize">
         <size>
          <width>321</width>
          <height>71</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>URW Chancery L</family>
          <pointsize>32</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>System  Everywhere</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="course_label">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>URW Bookman L</family>
              <pointsize>17</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Course</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="course">
            <property name="minimumSize">
             <size>
              <width>281</width>
              <height>61</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>URW Bookman L</family>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>course</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="schedule_label">
            <property name="minimumSize">
             <size>
              <width>131</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>URW Bookman L</family>
              <pointsize>17</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Schedule</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="schedule">
            <property name="minimumSize">
             <size>
              <width>281</width>
              <height>61</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>URW Bookman L</family>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>schedule</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="teacher_label">
            <property name="minimumSize">
             <size>
              <width>131</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>URW Bookman L</family>
              <pointsize>17</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Teacher</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="teacher">
            <property name="minimumSize">
             <size>
              <width>281</width>
              <height>61</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>URW Bookman L</family>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>teacher</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="feedback">
          <property name="minimumSize">
           <size>
            <width>290</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="function_label">
          <property name="minimumSize">
           <size>
            <width>141</width>
            <height>21</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>URW Bookman L</family>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="text">
           <string>Function</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="minimumSize">
           <size>
            <width>301</width>
            <height>270</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>268</height>
            </rect>
           </property>
           <widget class="QPushButton" name="btn1">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>101</width>
              <height>61</height>
             </rect>
            </property>
            <property name="text">
             <string>人脸签到</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn2">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>20</y>
              <width>101</width>
              <height>61</height>
             </rect>
            </property>
            <property name="text">
             <string>随机点名</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn3">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>100</y>
              <width>101</width>
              <height>61</height>
             </rect>
            </property>
            <property name="text">
             <string>函数绘图</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn4">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>100</y>
              <width>101</width>
              <height>61</height>
             </rect>
            </property>
            <property name="text">
             <string>白板</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn5">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>180</y>
              <width>101</width>
              <height>61</height>
             </rect>
            </property>
            <property name="text">
             <string>控制面板</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn6">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>180</y>
              <width>101</width>
              <height>61</height>
             </rect>
            </property>
            <property name="text">
             <string>备用</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_alarm">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string>打开时间提醒</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="time">
            <property name="minimumSize">
             <size>
              <width>181</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>2021-01-01 13:59</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>18</height>
           </size>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Entry</receiver>
   <slot>select_scene()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn1</sender>
   <signal>clicked()</signal>
   <receiver>Entry</receiver>
   <slot>face_register()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn2</sender>
   <signal>clicked()</signal>
   <receiver>Entry</receiver>
   <slot>random_register()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn3</sender>
   <signal>clicked()</signal>
   <receiver>Entry</receiver>
   <slot>myplot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn4</sender>
   <signal>clicked()</signal>
   <receiver>Entry</receiver>
   <slot>whiteBoard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn5</sender>
   <signal>clicked()</signal>
   <receiver>Entry</receiver>
   <slot>system_monitor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn6</sender>
   <signal>clicked()</signal>
   <receiver>Entry</receiver>
   <slot>standby()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_alarm</sender>
   <signal>clicked()</signal>
   <receiver>Entry</receiver>
   <slot>alarm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>select_scene()</slot>
  <slot>face_register()</slot>
  <slot>random_register()</slot>
  <slot>myplot()</slot>
  <slot>whiteBoard()</slot>
  <slot>system_monitor()</slot>
  <slot>standby()</slot>
  <slot>alarm()</slot>
 </slots>
</ui>
